app/lib/logic/provider_signup.rb

```diff
@@ -10,90 +10,6 @@ def signup_provider_possible?
         ensure_master
         !!(services.default && account_plans.default && services.default.service_plans.default)
       end
-
-      def signup_provider(application_plan = nil, options = {})
-        ensure_master
```

Comment:
I guess this is covered already at the level of each controller that now uses `Signup::ProviderAccountManager` instead of `Logic::.ProviderSignup#signup_provider`, correct?

Reply:
Now it is ensured by the controllers. Otherwise, it does not have permission for the required request.